package.json

```diff
@@ -127,6 +129,7 @@
     "migrate-mongo-ts": "^1.1.4",
     "mongoose": "^  5.13.15",
     "mongoose-softdelete-typescript": "^0.0.3",
+    "mongodb-memory-server": "^8.15.1",
```

Comment:
We shouldn't do this. This will download binaries we will never use to the docker image.

Reply:
We decided to follow-up on a separate MR